server/services/TransformService.ts

```diff
@@ -4,7 +4,7 @@
  */
 
 import {
-  IClusterClient,
+  ILegacyCustomClusterClient,
```

Comment:
LegacyClient and Client are fully compatible except the implementation?

Reply:
Yes.